src/org/omegat/gui/glossary/DictionaryGlossaryRenderer.java

```diff
@@ -57,8 +57,11 @@ public void render(GlossaryEntry entry, IRenderTarget<?> trg) {
         for (int i = 0; i < targets.length; i++) {
             if (i == 0 || (!targets[i].equals(targets[i - 1]))) {
                 if (hasComments) {
-                    trg.append("\n\t");
-                    hasComments = false;
+                    if (trg instanceof HtmlTarget) {
```

Comment:
This smells like a problem. We shouldn't have to know about the type of `trg`. The render target itself should know what to do here. Like maybe `IRenderTarget` should have methods like `startIndent` and `endIndent` or something?

Reply:
Yes I agree with that. We cannot be sure that there will not be another renderer in the future.
Two possibilities: either methods return a string and we write trg.append(trg.startIndent()) or we have a method trg.appendStartIndent(), as you prefer

Reply:
Add `trg.appendStartIndent()`